CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.4+5
```

Comment:
It's fine bumping it to 0.0.5 for new features and using the + for the small fixes.